Cargo.toml

```diff
@@ -6,7 +6,6 @@ members = [
 # We need to explicitly specify resolver 2.
 # We shouldn't have to per https://doc.rust-lang.org/edition-guide/rust-2021/default-cargo-resolver.html, but that does not appear to be the case
 resolver = "2"
-edition = "2021"
```

Comment:
Has 2021-edition been made default? Is 2015-edition desirable here, and that's why this config line was deleted?

Reply:
`edition` is only valid as a key for a package.  This means each package defines this value, but not the workspace as a whole.
Placing it in the workspace results in 
```shell
warning: .../git/sgx-std/Cargo.toml: unused manifest key: workspace.edition
```